llvm/include/llvm/CodeGen/LivePhysRegs.h

```diff
@@ -207,6 +209,18 @@ static inline bool recomputeLiveIns(MachineBasicBlock &MBB) {
   return oldLiveIns != newLiveIns;
 }
 
+/// Convenience function for recomputing live-in's for a set of MBBs until the
+/// computation converges.
+static inline void
```

Comment:
This new function doesn't need to be defined as `static` in the header file.

Reply:
Fixed, thanks.